R/dust_class.R

```diff
@@ -227,7 +227,7 @@ dust_class <- R6::R6Class(
     ##' @description
     ##' Returns the number of distinct data elements required. This is `0`
     ##' where the object was initialised with `data_multi = FALSE` and
-    ##' an integer otherwise.  For multi-data dust objects, Where `data`
+    ##' an integer otherwise.  For mulit-data dust objects, Where `data`
```

Comment:
```suggestion
    ##' an integer otherwise.  For multi-data dust objects, Where `data`
```
Merge mistake?

Reply:
This file is generated from one elsewhere, will track the source down